parkhangs-expressapp/models/EventModel.js

```diff
@@ -5,8 +5,9 @@ const Event = new Schema(
     {
         parkId: {type: String, required: true},
         details: {type: String, required: true},
-        eventDateTime: {type: String, required: true},
-        createdDateTime: {type: String, required: true},
+        eventDateTime: {type: String, required: true}, //when it starts
+        eventEndDateTime: {type: String, required: false}, //when it ends/ not required
```

Comment:
If end is not specified how does it show up on the calendar page?

Reply:
It doesn't need an end date to show up. It's using eventDateTime I believe. @cmaija can you confirm?

Reply:
yeah the calendar page uses eventDateTime but gordon you should be able to see if dates are showing up. Are they?

Reply:
yes, it is working